src/send.js

```diff
@@ -32,8 +35,14 @@ let sendDataPoints = async (configurer, options, payload) => {
       },
       encoding: null,
       body:     serialize(payload),
-      timeout:  30 * 1000,
-      retries:  5,
+      timeout:  90 * 1000,
+      retries:  (iter, error) => {
```

Comment:
I guess maybe 30s is safer... I just want this landed so we have fewer errors in sentry...